tests/unit/test_instance_subcmd.py

```diff
@@ -42,9 +45,9 @@
 Commands:
   associators   Get associated instances or names.
   count         Get instance count for classes.
-  create        Create an instance of classname.
+  create        Create an instance of CLASSNAME.
   delete        Delete a single CIM instance.
-  enumerate     Enumerate instances or names of CLASSNAME.
+  enumerate     Enumerate CIMInstances or names of CLASSNAME.
```

Comment:
Don't use class names in plural. Actually, to be consistent with the other commands, you could simply use "instances"

Reply:
Modified to say  instances.

Note: In this first sentence we need to keep it very short so we do not get help like:

enumerate  Enumerate CIM instances or names of . . .

And keep the full sentence on that single like of output.